pkgs/development/python-modules/jaxlib/default.nix

```diff
@@ -282,13 +282,7 @@ let
           --replace "/usr/bin/install_name_tool" "${cctools}/bin/install_name_tool"
         substituteInPlace ../output/external/rules_cc/cc/private/toolchain/unix_cc_configure.bzl \
           --replace "/usr/bin/libtool" "${cctools}/bin/libtool"
-      '' + (if stdenv.cc.isGNU then ''
-        sed -i 's@-lprotobuf@-l:libprotobuf.a@' ../output/external/org_tensorflow/third_party/systemlibs/protobuf.BUILD
-        sed -i 's@-lprotoc@-l:libprotoc.a@' ../output/external/org_tensorflow/third_party/systemlibs/protobuf.BUILD
-      '' else if stdenv.cc.isClang then ''
-        sed -i 's@-lprotobuf@${protobuf}/lib/libprotobuf.a@' ../output/external/org_tensorflow/third_party/systemlibs/protobuf.BUILD
-        sed -i 's@-lprotoc@${protobuf}/lib/libprotoc.a@' ../output/external/org_tensorflow/third_party/systemlibs/protobuf.BUILD
-      '' else throw "Unsupported stdenv.cc: ${stdenv.cc}");
```

Comment:
why is this no longer necessary?

Reply:
When trying to build jaxlib from source I faced a build failure at the `preBuild` phase.
More precisely, linking the protobuf lib in `../output/external/org_tensorflow` was failing because this folder doesn't exist.
I removed this patch and it built successfully.

It seems to come from [this upstream commit](https://github.com/google/jax/commit/172a831219aa7d3524c0c8b5779dc29597a05810).

Reply:
Ah makes sense... I didn't realize they had renamed tensorflow_org -> xla